core/profiles/standard/standard.install

```diff
@@ -191,7 +191,7 @@ function standard_install() {
   config('path.settings')
     ->set('node_page_pattern', '[node:title]')
     ->set('node_post_pattern', 'posts/[node:title]')
-    ->set('node_card_pattern', 'cards/[node:title]')
+    ->set('node_card_pattern', 'node/[node:nid]')
```

Comment:
If we're going to remove the pattern, we should simply delete this line. Otherwise we'll just be making aliases to the original path, a confusing and unnecessary alias.

Reply:
@quicksketch - removing that line does nothing. We have a default path pattern for all new content types. 

Line 193 does nothing, neither does line 194. @jenlampton suggested we remove line 194 to remove the pattern, but I left it in the PR because removing it does nothing except create an inconsistency in how we are treating posts and cards. 

To the best of my knowledge, to remove the default pattern for cards we either need to:

1) remove the default pattern for all content types OR
2) override as I did in this PR

Reply:
Unless you have a better idea.

Reply:
I think line 193 effectively changes the pattern from "post/[node:title]" to "posts/[node:title]" (pluralizing the path).

But you're right that because we have a default node pattern, all nodes will get a pattern even if we remove this line. I tested this, and sure enough, no new alias gets created (though I'm not sure how it works that way). It seems to work, so I retract my comment. I don't know of any other way to prevent the default from taking effect.